vendor/parser-extended/parsetree.mli

```diff
@@ -1361,6 +1366,22 @@ and directive_argument_desc =
   | Pdir_ident of Longident.t
   | Pdir_bool of bool
 
+(** Lexer directives: ugly hack to avoid their deletion *)
+and syntax_directive =
+  {
+    psyn_mode: string loc;
+    psyn_toggle: bool;
```

Comment:
It's not obvious to me what `psyn_mode` and `psyn_toggle` represent - maybe a comment?